onnxruntime/core/graph/model.h

```diff
@@ -26,8 +26,8 @@ class Model {
   explicit Model(const std::string& graph_name,
                  bool is_onnx_domain_only,
                  const logging::Logger& logger)
-                 :Model(graph_name,is_onnx_domain_only, ModelMetaData(),IOnnxRuntimeOpSchemaRegistryList(),{},{},
-                     logger){}
+      : Model(graph_name, is_onnx_domain_only, ModelMetaData(), IOnnxRuntimeOpSchemaRegistryList(), {}, {},
```

Comment:
>{}, {} [](start = 100, length = 6)

I suggest you avoid {} initializers for the sake of old compilers and call default __ctor explicitly.

Reply:
Sure.

Reply:
Actually this seems like a formatting change. I would split refinements to existing code to another PR. Refining existing code that doesn't align with the theme of this PR doesn't seem too clean. I recall that we have some general complaints of the existing Model class (usage of unnecessary shared_ptr, etc.). Shall we tackle all these together ?